src/strikethrough.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license Copyright (c) 2017, CKSource - Rémy Hubscher. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * @module basic-styles/strikethrough
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import StrikethroughEngine from './strikethroughengine';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import strikethroughIcon from '../theme/icons/strikethrough.svg';
+
+/**
+ * The strikethrough feature. It introduces the Strikethrough button and the <kbd>Ctrl+Shift+X</kbd> keystroke.
+ *
+ * It uses the {@link module:basic-styles/strikethroughengine~StrikethroughEngine strikethrough engine feature}.
+ *
+ * @extends module:core/plugin~Plugin
+ */
+export default class Strikethrough extends Plugin {
+	/**
+	 * @inheritDoc
+	 */
+	static get requires() {
+		return [ StrikethroughEngine ];
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	static get pluginName() {
+		return 'Strikethrough';
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	init() {
+		const editor = this.editor;
+		const t = editor.t;
+		const command = editor.commands.get( 'strikethrough' );
+		const keystroke = 'CTRL+SHIFT+X';
+
+		// Add strikethrough button to feature components.
+		editor.ui.componentFactory.add( 'strikethrough', locale => {
+			const view = new ButtonView( locale );
+
+			view.set( {
+				label: t( 'Strikethrough' ),
+				icon: strikethroughIcon,
+				keystroke,
```

Comment:
It our version the tooltip does not show "SHIFT" in the tooltip?  is this also affected by this?

![image](https://user-images.githubusercontent.com/128755/33050640-ebc82fb8-ce33-11e7-810e-3c371a32cb8e.png)

Reply:
@vladikoff I check that out and I found out it's a bug https://github.com/ckeditor/ckeditor5-utils/issues/206.

Reply:
@szymonkups Could you check if https://github.com/ckeditor/ckeditor5-utils/pull/207 fixes the issue?

Reply:
@oleq - checked, tested and merged :)